app/uk/gov/hmrc/merchandiseinbaggage/repositories/DeclarationJourneyRepository.scala

```diff
@@ -52,13 +53,16 @@ class DeclarationJourneyRepository @Inject()(mongo: MongoComponent, @Named("decl
 
   private[repositories] lazy val indices = collection.listIndexes().toFuture()
 
-  def insert(declarationJourney: DeclarationJourney): Future[DeclarationJourney] =
-    collection.insertOne(declarationJourney).toFuture().map(_ => declarationJourney)
+  def insert(declarationJourney: DeclarationJourney): Future[DeclarationJourney] = {
+    val datedDeclarationJourney = Json.toJsObject(declarationJourney) ++
+      Json.obj("createdAtDate" -> Json.obj("$date" -> Instant.now.toEpochMilli))
```

Comment:
need to revert appending this date field here more likely we need to add the field in `DeclarationJourney` object and default the new field createdAtDate to `now` and it will be easier to work here and everywhere in the code